app/src/test/java/jp/developer/retia/frozenword/TestExt.kt

```diff
@@ -0,0 +1,41 @@
+package jp.developer.retia.frozenword
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import kotlinx.coroutines.test.resetMain
+import kotlinx.coroutines.test.setMain
+import org.spekframework.spek2.dsl.LifecycleAware
+
+@ExperimentalCoroutinesApi
+fun LifecycleAware.setMainDispatcher(dispatcher: TestCoroutineDispatcher = TestCoroutineDispatcher()) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/verno3632/FrozenWord/blob/5d531f90da2a5c007e00fca4d5f9537c4641fdd2/app/src/test/java/jp/developer/retia/frozenword/TestExt.kt#L12"></span>Parameter should be on a separate line (unless all parameters can fit a single line)</td>
    </tr>
  </tbody>
</table>